ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-ec.yaml

```diff
@@ -0,0 +1,624 @@
+base_images:
+  ansible:
+    name: "4.16"
+    namespace: ocp
+    tag: ansible
+  aws-efs-csi-operator-create-efs:
+    name: "4.16"
+    namespace: ocp
+    tag: aws-efs-csi-operator-create-efs
+  cli:
+    name: "4.16"
+    namespace: ocp
+    tag: cli
+  cli-ocm:
+    name: cli-ocm
+    namespace: ci
+    tag: latest
+  dev-scripts:
+    name: test
+    namespace: ocp-kni
+    tag: dev-scripts
+  hypershift-operator:
+    name: "4.16"
+    namespace: ocp
+    tag: hypershift-operator
+  openstack-installer:
+    name: "4.16"
+    namespace: ocp
+    tag: openstack-installer
+  rosa-aws-cli:
+    name: rosa-aws-cli
+    namespace: ci
+    tag: latest
+  tests-private:
+    name: tests-private
+    namespace: ci
+    tag: "4.16"
+  tests-private-postupg:
+    name: tests-private
+    namespace: ci
+    tag: "4.16"
+  tests-private-preupg:
+    name: tests-private
+    namespace: ci
+    tag: "4.16"
+  tests-private-ui:
+    name: tests-private-ui
+    namespace: ci
+    tag: "4.16"
+  tools:
+    name: "4.16"
+    namespace: ocp
+    tag: tools
+  upi-installer:
+    name: "4.16"
+    namespace: ocp
+    tag: upi-installer
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.18
+releases:
+  latest:
+    prerelease:
+      architecture: amd64
+      product: ocp
+      version_bounds:
+        lower: 4.16.0-0
```

Comment:
Just curious can we use below settings to narrow down the scope:
 lower: 4.16.0-ec
 upper: 4.16.0-rc

As I know the version order is:
0.nightly < stable < ec < rc

Reply:
What's the benefit of using `4.16.0-ec` compared with `4.16.0-0` ?

Reply:
I think `4.16.0-0` will include nightly if `4-dev-preview` include nightly build, like: 4.16.0-0.nightly-2024-02-02-002725

Reply:
![image](https://github.com/openshift/release/assets/83053288/8552b494-03cf-449d-a366-fb79ed7633e2)

Reply:
Have you ever seen nightly build exist in `4-dev-preview` ?

Reply:
I don't know that, so I said `I curious`

Reply:
From what I know, that issue/curious does not happen till now after we have `4-dev-preview`.